applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/settings.js

```diff
@@ -0,0 +1,217 @@
+
+	o = s.option(form.Value, 'sta_block_timeout', _('sta block timeout reports'), _('Maximum amount of time (ms) a station may be blocked due to policy decisions'));
+	o.optional    = true;
+	o.placeholder = 30000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'local_sta_timeout', _('local sta timeout'), _('Maximum amount of time (ms) a local unconnected station is tracked'));
+	o.optional    = true;
+	o.placeholder = 12000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'measurement_report_timeout', _('measurement report timeout'), _('Maximum amount of time (ms) a measurement report is stored'));
+	o.optional    = true;
+	o.placeholder = 120000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'local_sta_update', _('local_sta_update'), _('Local station information update interval (ms)'));
+	o.optional    = true;
+	o.placeholder = 1000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'max_retry_band', _('max retry band'), _('Maximum number of consecutive times a station may be blocked by policy'));
+	o.optional    = true;
+	o.placeholder = 5;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'seen_policy_timeout', _('seen policy timeout'), _('Maximum idle time of a station entry (ms) to be considered for policy decisions'));
+	o.optional    = true;
+	o.placeholder = 30000;
+	o.datatype    = 'uinteger';
+	
+	o = s.option(form.Value, 'load_balancing_threshold', _('load_balancing_threshold'), _('Minimum number of stations delta between APs before load balancing policy is active'));
+	o.optional    = true;
+	o.placeholder = 5;
+	o.datatype    = 'uinteger';
+	
+	o = s.option(form.Value, 'band_steering_threshold', _('band_steering_threshold'), _('Minimum number of stations delta between bands before band steering policy is active'));
+	o.optional    = true;
+	o.placeholder = 5;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'band_steering_threshold', _('band_steering_threshold'), _('Minimum number of stations delta between bands before band steering policy is active'));
+	o.optional    = true;
+	o.placeholder = 5;
+	o.datatype    = 'uinteger';	
+	
+	o = s.option(form.Value, 'remote_update_interval', _('remote_update_interval'), _('Interval (ms) between sending state updates to other APs'));
+	o.optional    = true;
+	o.placeholder = 1000;
+	o.datatype    = 'uinteger';	
+
+	o = s.option(form.Value, 'remote_node_timeout', _('remote_node_timeout'), _('Number of remote update intervals after which a remote-node is deleted'));
+	o.optional    = true;
+	o.placeholder = 10;
+	o.datatype    = 'uinteger';	
+
+    o = s.option(form.Flag, 'assoc_steering', _('assoc_steering'), _('Allow rejecting assoc requests for steering purposes (0/1)'));
+    o.optional    = true;
+	o.placeholder = 0;
+	
+	o = s.option(form.Value, 'min_connect_snr', _('min_connect_snr'), _('Minimum signal-to-noise ratio or signal level (dBm) to allow connections'));
+	o.optional    = true;
+	o.placeholder = 0;
```

Comment:
no need for 0 placeholder.  Is that even an actually useful value here?

Reply:
The zero is the default value if you leave it blank (according to the wiki page). That is the reason I put in all the defaults to indicate what the defaults are.